internal/kafka/command_topic_consume.go

```diff
@@ -50,8 +53,10 @@ func (c *command) newConsumeCommand() *cobra.Command {
 	cmd.Flags().Bool("timestamp", false, "Print message timestamp in milliseconds.")
 	cmd.Flags().StringSlice("config", nil, `A comma-separated list of configuration overrides ("key=value") for the consumer client.`)
 	pcmd.AddConsumerConfigFileFlag(cmd)
-	cmd.Flags().String("schema-registry-context", "", "The Schema Registry context under which to look up schema ID.")
 	cmd.Flags().String("schema-registry-endpoint", "", "Endpoint for Schema Registry cluster.")
+
+	// cloud-only flags
```

Comment:
In some other commands we use `cfg.IsCloudLogin()` to determine which flags to show, is that something we could do here as well?

Reply:
We want this command to be able to be run without requiring login, for some reason... it was a customer request.

Reply:
Ah, right.

Reply:
not really. The only other command that shows both flags is `login`, but you haven't logged in yet so no